nemo_aligner/algorithms/ppo.py

```diff
@@ -42,6 +47,82 @@
 from nemo_aligner.utils.utils import clear_memory, cpu_dict, masked_mean
 
 
+class PPORolloutBatch(UserDict):
```

Comment:
yeah good point. Really I'm just afraid of inheriting from python primitive types since they seem so magical. But I find it more explicit than inheriting from `dict` directly.

On the UserDict docs it actually says the following:
```
The class, [UserDict](dfile:///Users/geshen/Library/Application%20Support/Dash/DocSets/Python_3/Python%203.docset/Contents/Resources/Documents/doc/library/collections.html#collections.UserDict) acts as a wrapper around dictionary objects. The need for this class has been partially supplanted by the ability to subclass directly from [dict](dfile:///Users/geshen/Library/Application%20Support/Dash/DocSets/Python_3/Python%203.docset/Contents/Resources/Documents/doc/library/stdtypes.html#dict); however, this class can be easier to work with because the underlying dictionary is accessible as an attribute.
```

Reply:
Ok not a big deal, feel free to resolve